### DISCUSSION TITLE
服务器有人进入就会断联

### DISCUSSION THREAD

**Discussion by cql311** [natfrp/wiki#446] [使用问题]
求助，之前一直没有出现这个问题，但是出现之后这个问题经常存在，切换网络、关闭防火墙都没用
![Uploading 1708604693876.jpg…]()

**Comment by cql311** [comment]
![1708604693876](https://github.com/natfrp/wiki/assets/160861135/6a00c8a2-2704-4220-9a49-3790a50d08cc)

**Comment by fakeboboliu** [comment]
此问题理论上与我们无关，请检查你的 192.168.101.1:25566 服务的日志。

_我没看懂这是搞了个啥。_
